core/spec/models/spree/order_merger_spec.rb

```diff
@@ -28,8 +28,12 @@ module Spree
 
     context 'merging together two orders with line items for the same variant' do
       before do
-        order_1.contents.add(variant, 1)
-        order_2.contents.add(variant, 1)
+        Spree::Dependencies.cart_add_item_service.constantize.call(order: order_1,
```

Comment:
@amalrik we can just use `Spree::Cart::AddItem` here - no need to use dependencies in specs - thanks!